src/runtime/audience-action-flow-test.js

```diff
@@ -27,8 +27,10 @@ import {Constants} from '../utils/constants';
 import {PageConfig} from '../model/page-config';
 import {ProductType} from '../api/subscriptions';
 import {Toast} from '../ui/toast';
+import {nowAsSecondsString} from '../utils/date-utils';
 
 const WINDOW_LOCATION_DOMAIN = 'https://www.test.com';
+const CURRENT_TIME = 1615416442; // GMT: Wednesday, March 10, 2021 10:47:22 PM
```

Comment:
Weird.. documentation suggests this should be `1615416442000` for this specific date

Reply:
Yeah weird. I copied it from another place in the code, but the date doesn't really matter so I'll just remove it.